src/components/TextField/TextField.module.scss

```diff
@@ -252,7 +253,7 @@
 }
 
 .textField .inputWrapper.wrapperSizeMedium .iconContainer .icon {
-  font-size: var(--font-size-paragraph);
+  font-size: 16px;
```

Comment:
Same

Reply:
As discussed we deiced to keep it because it's a font size for font icon and not text